analysis/tvla.py

```diff
@@ -585,17 +614,22 @@ def run_tvla(ctx: typer.Context):
                 else:
                     leakage = np.load(cfg["leakage_file"])
                     assert num_traces == leakage.shape[2]
-            else:
+            elif general_test_key:
                 log.info("Computing Leakage")
                 # We identify the fixed key by looking at the first 20 keys in the project.
-                leakage = compute_leakage_general(keys, find_fixed_key(keys_nparrays[0:20]))
+                leakage = compute_leakage_general(keys, find_fixed_entry(keys_nparrays[0:20]))
+            else:
+                assert general_test_data
+                log.info("Computing Leakage")
+                # We identify the fixed key by looking at the first 20 plaintexts in the project.
```

Comment:
Nit: "We identify the fixed data..."

Reply:
Thanks for catching this. It is fixed now.